tests/utils/gen_hugging_face_text_classification.py

```diff
@@ -0,0 +1,47 @@
+import json
+import os
+import sys
+
+import requests
+from kili.client import Kili
+
+if __name__ == "__main__":
+    project_id = sys.argv[1]
+
+    k = Kili()
+    g = k.assets(project_id=project_id, as_generator=True)
+
+    assets = []
+    for i, a in enumerate(g):
+        assets.append(a)
+        if i > 50:
+            break
+
+    for a in assets:
+        for label in a["labels"]:
+            del label["author"]
```

Comment:
This should be in a command  refresh_mock_data.py which would create the mock jsons